app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -3,4 +3,11 @@ class ManageIQ::Providers::Google::Inventory::Parser < ManageIQ::Providers::Inve
   require_nested :NetworkManager
 
   VENDOR_GOOGLE = "google".freeze
+
+  def parse_uid_from_url(url)
+    # A lot of attributes in gce are full URLs with the
+    # uid being the last component.  This helper method
+    # returns the last component of the url
+    url.split('/')[-1]
+  end
```

Comment:
This was the only method left in the RefreshHelperMethods mixin after removing ems_inv_to_hashes and both the cloud_manager and network_manager inherit from this parser base class so I just moved it into here.